### DISCUSSION TITLE
Does the build process make use of npm's cache or does it require a fresh install each time?

### DISCUSSION THREAD

**Discussion by andyjessop** [vercel/vercel#7885] [Help]
Let's say I want to run a script for the "Ignored Build Step" feature, and that script requires an `npm install`. Will the platform retrieve the result from cache? Or will it install everything from source on each build?

The reason I'm asking is that if it takes a few minutes to run this command then it dramatically slows down our workflow if several developers are building concurrently.

**Comment by oldbrush** [comment]
@andyjessop https://vercel.com/changelog/faster-build-times-for-monorepos
